health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppConfigurationWrapper.js

```
@@ -0,0 +1,705 @@
+                  type: "ADD_FIELD",
+                  payload: {
+                    ...popupData,
+                    fieldData: addFieldData,
+                  },
+                });
+                setPopupData(null);
+                setAddFieldData(null);
+              }}
+            />,
+          ]}
+        ></PopUp>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)